docs/guide/assets.md

```diff
@@ -115,5 +115,8 @@ This pattern does not work if you are using Vite for Server-Side Rendering, beca
 :::
 
 ::: warning Esbuild target config is necessary
+
+<!-- mark -->
+
 This pattern needs esbuild target to be set to `es2020` or higher. `vite@2.x` use `es2019` as default target. Set [build-target](https://vitejs.dev/config/#build-target) and [optimizedeps.esbuildoptions.target](https://vitejs.dev/config/#optimizedeps-esbuildoptions) to `es2020` or higher if you intend to use this partten.
```

Comment:
Is this still valid?

Reply:
Default target will be changed in #7644. So after that has landed, the description will be outdated.

Reply:
So I will wait with this PR until this other PR was merged?

Reply:
Maybe this part of the document should be updated in #7644.

Reply:
We actually also have to update prebundling to fix to `es2020` instead of `config.build.target`

https://github.com/vitejs/vite/blob/e0fe2008b248aec4813efba34db9e644b851943a/packages/vite/src/node/optimizer/index.ts#L459

That change should come after #7644